app/src/main/java/com/android/partagix/model/FinishedLoansViewModel.kt

```diff
@@ -0,0 +1,69 @@
+package com.android.partagix.model
+
+import android.location.Location
+import androidx.lifecycle.ViewModel
+import com.android.partagix.model.auth.Authentication
+import com.android.partagix.model.category.Category
+import com.android.partagix.model.item.Item
+import com.android.partagix.model.loan.Loan
+import com.android.partagix.model.loan.LoanState
+import com.android.partagix.model.visibility.Visibility
+import java.util.concurrent.CountDownLatch
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+
+class FinishedLoansViewModel(db: Database = Database(), latch: CountDownLatch = CountDownLatch(1)) :
```

Comment:
the latch isnt use in the tests if nobody is waiting for the latchcountdown you can delete the latch and the latch countdown

Reply:
Nice catch thanks !